lib/cinch/test.rb

```diff
@@ -54,10 +54,11 @@ def initialize(msg, bot, opts = {})
         @user = Cinch::User.new(opts.delete(:nick) { 'test' }, bot)
         if opts.key?(:channel)
           @channel = Cinch::Channel.new(opts.delete(:channel), bot)
+          @target = @channel
         end
 
-        # set the message target
-        @target = @channel || @user
+        # message target is channel if it exists (above), else it's user
+        @target ||= @user
```

Comment:
oh, I mainly put the `||=` here because I wanted to deal with the case where `@channel` is nil despite `opts.key?(:channel)` being true, but that actually never happens, so in fact this could be put in an `else` under the `if`. In fact I think I'll go do that right now...